src/meta/src/barrier/mod.rs

```diff
@@ -211,17 +212,17 @@ where
         if self.enable_recovery {
             // handle init, here we simply trigger a recovery process to achieve the consistency. We
             // may need to avoid this when we have more state persisted in meta store.
-            let new_epoch = self.env.epoch_generator().generate().into_inner();
+            let new_epoch = Epoch::now().0;
             assert!(new_epoch > state.prev_epoch);
             state.prev_epoch = new_epoch;
 
             let (new_epoch, actors_to_finish, finished_create_mviews) =
                 self.recovery(state.prev_epoch, None).await;
-            unfinished.add(new_epoch.into_inner(), actors_to_finish, vec![]);
+            unfinished.add(new_epoch.0, actors_to_finish, vec![]);
             for finished in finished_create_mviews {
                 unfinished.finish_actors(finished.epoch, once(finished.actor_id));
             }
-            state.prev_epoch = new_epoch.into_inner();
+            state.prev_epoch = new_epoch.0;
             state.update(self.env.meta_store()).await.unwrap();
```

Comment:
It's okay but a little bit confused to use `new_epoch` as u64 at first then use it as `Epoch` and there is no relation between the former and the later one.

Reply:
There's too many places using different type of Epoch and u64. Tried to fix some already.